Rocket.Chat/Controllers/Auth/ConnectServerViewController.swift

```diff
@@ -121,6 +121,8 @@ final class ConnectServerViewController: BaseViewController {
             text = defaultURL
         }
 
+        API.shared.host = text
```

Comment:
This can't be used this way, `text` may not have the protocol.

Reply:
You're right! I'll find a good way to handle this soon.